### PR TITLE
Feature/add aria label to interactive legend buttons

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard22**
Added an aria label to interactive legends which use age. Screen readers were not able to let the user know what unit the number were in.

### CODE REVIEW

File: packages/app/src/components/interactive-legend.tsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ export interface SelectOption<T = string> {
   label: string;
   color: string;
   shape?: 'line' | 'circle' | 'square';
+  ariaLabel?: 'string';
```
Comment (VWSCoronaDashboard14) [inline packages/app/src/components/interactive-legend.tsx:14]: ` ariaLabel?: 'string';` -> `ariaLabel?: string;`

Comment (VWSCoronaDashboard1) [inline packages/app/src/components/interactive-legend.tsx:14]:
```suggestion
  legendAriaLabel?: string;
```

**Review by VWSCoronaDashboard1 — CHANGES_REQUESTED** [review]
Are you 100% sure that those are the only charts with interactive legends? I thought there were more, but I could be mistaken...

File: packages/app/src/components/interactive-legend.tsx (hunk 1 above)
Comment (VWSCoronaDashboard22) [reply packages/app/src/components/interactive-legend.tsx]: Fixed SelectOption interace and renamed ariaLabel property

**Review by VWSCoronaDashboard1 — APPROVED** [review]

**Review by VWSCoronaDashboard14 — APPROVED** [review]

**Pull request merged**